migrations/postgres/20230505071708_remove_legacy_tables.up.sql

```diff
@@ -0,0 +1,3 @@
+DROP TABLE physical_partition_column;
```

Comment:
Kinda scary to just delete these. How about, before we run the DB migrations, we check if the user still has some data in these tables and if so, output an error message ("make sure to use Seafowl 0.3 to run the Seafowl 0.3 -> 0.4 migration process") and exit (could even make a final final 0.3 release and do this migration process automatically)?

Reply:
Agreed, that makes sense. I've now added some checks to this effect (basically if there are any legacy tables or partitions we won't run 0.4 migrations), and a couple of tests as well.